librz/asm/arch/cil/cil_dis.c

```diff
@@ -0,0 +1,158 @@
+// SPDX-FileCopyrightText: 2022 wingdeans
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_types.h>
+#include <rz_asm.h>
+#include <rz_endian.h>
+#include <rz_util/rz_strbuf.h>
+#include "cil_dis.h"
+
+// The read_## functions are dispatched based on the opcode `param`
```

Comment:
I recommend using the Doxygen format for comments, wherever possible, e.g. in this case it would be `\file`.